spatialos-sdk/Cargo.toml

```diff
@@ -13,6 +13,7 @@ spatialos-sdk-sys = { path = "../spatialos-sdk-sys"}
 futures = "0.1"
 inventory = "0.1"
 lazy_static = "1.3"
+static_assertions = "1.1.0"
```

Comment:
Personally prefer marking this as `dev-dependency` and wrapping the assertions with `#[cfg(test)]` so as to eliminate a dependency for an end user.

Since we run `cargo test` in CI, we will always check these assertions as a pre-merge check

Reply:
Ah, good idea! 😁